src/Service/Startup.cs

```diff
@@ -79,10 +79,10 @@ public void ConfigureServices(IServiceCollection services)
                 null);
             IFileSystem fileSystem = new FileSystem();
             FileSystemRuntimeConfigLoader configLoader = new(fileSystem, configFileName, connectionString);
+            services.AddSingleton<IOpenApiDocumentor, OpenApiDocumentor>();
```

Comment:
Add a check to make sure we do this only when Rest is enabled. ref:  https://github.com/Azure/data-api-builder/pull/1872/files